src/metabase/models/pulse.clj

```diff
@@ -192,7 +193,7 @@
   "Update an existing `Pulse`, including all associated data such as: `PulseCards`, `PulseChannels`, and `PulseChannelRecipients`.
 
    Returns the updated `Pulse` or throws an Exception."
-  [{:keys [id name cards channels] :as pulse}]
+  [{:keys [id name cards channels skip_if_empty?] :as pulse}]
```

Comment:
Arguably the other argument to that function `creator_id` should actually be `creator-id` as well. But that actually comes back from the DB that way. If you wanted to change things for consistency than you could match what comes back from the DB (`skip_if_empty`) or fix them both to be `lisp-case` (`creator-id` and `skip-if-empty`) but since this is test code it's not particularly important.

But either way `skip_if_empty?` is neither something that comes back from the DB/API or something in proper `lisp-case` so it doesn't make sense